apps/web/pages/success.tsx

```diff
@@ -298,7 +321,13 @@ export default function Success(props: inferSSRProps<typeof getServerSideProps>)
                               .format("YYYYMMDDTHHmmss[Z]")}&text=${eventName}&details=${
                               props.eventType.description
                             }` +
-                            (typeof location === "string" ? "&location=" + encodeURIComponent(location) : "")
+                            (typeof location === "string"
+                              ? "&location=" + encodeURIComponent(location)
+                              : "") +
+                            (props.eventType.recurringEvent
+                              ? "&recur=" +
+                                encodeURIComponent(new rrule(props.eventType.recurringEvent).toString())
+                              : "")
```

Comment:
Adding the configuration to the Google Calendar link on the success page when booking a recurring event. See how easy it is, thanks `rrule`! BTW, could find a way to do the same in the other calendars, created a bounty in Stackoverflow just in case there is an obscured way to do it.